src/Tools/NodeDocumentationMarkdownGenerator/Commands/RenameCommand.cs

```diff
@@ -75,6 +103,29 @@ private static void RenameDirectory(string directory, int maxLength)
             }
 
             var allMdFiles = Directory.GetFiles(directory, "*.md", SearchOption.TopDirectoryOnly).Select(x => new FileInfo(x)).ToList();
+            var allFiles = Directory.GetFiles(directory,"*",SearchOption.TopDirectoryOnly).Select(x => new FileInfo(x)).ToList();
+            var allNonMDFiles = allFiles.Except(allMdFiles);
+            if (Verbose)
+            {
+                //do a scan for support files which do not share the base name.
+                foreach (var nonMDFile in allNonMDFiles)
```

Comment:
So you did find support files without a base file? What that caused by someone deleting the basename but not the supportfiles? I can't remember seeing this before.

Reply:
there is a single support file that is referenced from one of the renamed files, but its argument names are different, so it does not get renamed.

See my sample output for the example.

I didn't end up changing it as the name seemed short enough.